spec/std/thread/condition_variable_spec.cr

```diff
@@ -1,90 +1,89 @@
 require "spec"
 
-# These specs are marked as pending until we add
-# support for parallelism, where we'll see if we
-# need condition variables.
```

Comment:
Since these were pending until somebody works on parallelism, and nobody did that yet, is it worth uncommenting these?

Reply:
Who said "nobody's working on MT" :)

But sure, I can keep the note.

Reply:
Oh, I meant all the specs. But if "somebody" ;-) is working on it, then this is fine.